package/docker/Dockerfile.ubuntu

```diff
@@ -5,13 +5,41 @@ ARG BASE_IMAGE=focal
 ENV TZ=America/Chicago
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
-RUN    apt-get update                   \
-    && apt-get upgrade --yes            \
-    && apt-get install --yes            \
-                        build-essential \
-                        git             \
-                        python          \
-                        python3-pip
+RUN    apt-get update                                   \
+    && apt-get install --yes software-properties-common \
+    && add-apt-repository ppa:ethereum/ethereum
+
+RUN    apt-get update            \
+    && apt-get upgrade --yes     \
+    && apt-get install --yes     \
```

Comment:
If I understand correctly, the goal is to make `solc-to-k` work in the container. Is this the minimal list of dependencies to achieve that?